prow/plugins/blockade/blockade.go

```diff
@@ -74,25 +74,9 @@ func helpProvider(config *plugins.Configuration, enabledRepos []config.OrgRepo)
 		}
 		blockConfig[repo.String()] = buf.String()
 	}
-	yamlSnippet, err := plugins.CommentMap.GenYaml(&plugins.Configuration{
-		Blockades: []plugins.Blockade{
-			{
-				Repos: []string{
-					"ORGANIZATION",
-					"ORGANIZATION/REPOSITORY",
-				},
-				BlockRegexps: []string{
-					"^examples*",
-				},
-				ExceptionRegexps: []string{
-					"^examples-keep/",
-				},
-				Explanation: "examples have moved elsewhere",
-			},
-		},
-	})
```

Comment:
Why is this removed? The description does not mention it.

Reply:
Good catch... I did wrong with my rebase.